config/webhook.yaml

```diff
@@ -56,6 +56,10 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        securityContext:
+          runAsUser: 1337
+          runAsGroup: 1337
+          allowPrivilegeEscalation: false
```

Comment:
I think I'd rather see a `:nonroot` distroless base image released (or change the default image) and use that.

We can override the default base image via `.ko.yaml` in the repo root.

Reply:
cc @dlorenc who I was asking about this recently.

Reply:
AFAIU, if we do the nonroot image approach then we would still want `runAsNonRoot: true` and `allowPrivilegeEscalation: false` in the security context.

Reply:
I'd rather see use of images that did not run as root than explicitly setting these fields.